.github/workflows/tests.yml

```diff
@@ -218,6 +212,14 @@ jobs:
           - python-version: "3.9"
             runtime-version: "latest"
             os: "macos-latest"
+          - python-version: "3.7"
```

Comment:
Did you mean 3.8, or we are still testing 3.7 against 0.0.3 only for ubuntu?

Reply:
I see that makes sense, that was my only comment/question. I see CI is green so I'll go ahead and merge this one.